src/modules/app/components/AppLayoutContent.tsx

```
@@ -0,0 +1,74 @@
+        window.clearTimeout(closeTimeoutRef.current);
+      }
+    };
+  }, []);
+
+  return (
+    <>
+      {!isSidebarOpen && !isMobile && (
+        <div
+          className="fixed top-0 left-0 z-30 h-full w-4"
+          onMouseEnter={handleMouseEnter}
+        />
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)